Buildings/Controls/OBC/FDE/DOAS/EnergyWheel.mo

```diff
@@ -0,0 +1,217 @@
+        iconTransformation(extent={{-122,6},{-82,46}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput ecoMode
+    "True when economizer mode is active."
+      annotation (Placement(transformation(extent={{-142,30},{-102,70}}),
+        iconTransformation(extent={{-128,30},{-88,70}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput erwT
+    "Energy recovery wheel supply air temperature."
+      annotation (Placement(transformation(extent={{-142,-74},{-102,-34}}),
+        iconTransformation(extent={{-124,-74},{-84,-34}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput supPrimSP
+    "Primary supply air temperature set point."
+      annotation (Placement(transformation(extent={{-142,-106},{-102,-66}}),
+        iconTransformation(extent={{-122,-108},{-82,-68}})));
+
+    // ---outputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput erwStart
+    "Command to start the energy recovery wheel."
+      annotation (Placement(transformation(extent={{102,0},{142,40}}),
+        iconTransformation(extent={{86,-12},{126,28}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput erwSpeed
+    "Energy recovery wheel speed command."
+      annotation (Placement(transformation(extent={{102,-96},{142,-56}}),
+        iconTransformation(extent={{82,-96},{122,-56}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput bypDam
+    "Bypass damper command; true when commanded full open."
+      annotation (Placement(transformation(extent={{102,46},{142,86}}),
+        iconTransformation(extent={{84,46},{124,86}})));
+
+  Buildings.Controls.OBC.CDL.Continuous.Add difference(
+    final k2=-1)
+      annotation (Placement(transformation(extent={{-90,-10},{-70,10}})));
+  Buildings.Controls.OBC.CDL.Continuous.Abs abs
+    annotation (Placement(transformation(extent={{-62,-10},{-42,10}})));
+  Buildings.Controls.OBC.CDL.Continuous.Greater gre
+    annotation (Placement(transformation(extent={{-26,-10},{-6,10}})));
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant conRecovery(
+    k=recSet)
+      "Recovery set point."
+        annotation (Placement(transformation(extent={{-62,-46},{-42,-26}})));
+  Buildings.Controls.OBC.CDL.Logical.Latch lat
+    annotation (Placement(transformation(extent={{30,-10},{50,10}})));
+  Buildings.Controls.OBC.CDL.Continuous.Less les
+    annotation (Placement(transformation(extent={{-26,-38},{-6,-18}})));
+  Buildings.Controls.OBC.CDL.Logical.TrueDelay truDel(
+    delayTime=recSetDelay,
+    delayOnInit=true)
+      "Recovery set point delay before disabling energy wheel."
+        annotation (Placement(transformation(extent={{2,-38},{22,-18}})));
+  Buildings.Controls.OBC.CDL.Logical.And3 and3
+    annotation (Placement(transformation(extent={{62,10},{82,30}})));
+  Buildings.Controls.OBC.CDL.Logical.Not not1
+    annotation (Placement(transformation(extent={{-26,40},{-6,60}})));
+  Buildings.Controls.OBC.CDL.Continuous.LimPID conPID(
```

Comment:
One PID controller would be enough. Also, the controller output would need to be reset when the ERW enabled.

Reply:
The current CDL PID controller can only calculate when measured value is either above set point or below set point based on 'reverseAction' parameter, not both simultaneously. This logic needs to calculate when the measured value is away from set point regardless of direction.